src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/services.component.spec.ts

```diff
@@ -79,7 +79,11 @@ describe('ServicesComponent', () => {
   });
 
   it('should have columns that are sortable', () => {
-    expect(component.columns.every((column) => Boolean(column.prop))).toBeTruthy();
+    expect(
+      component.columns
+        .filter((column) => !(column.prop === undefined))
```

Comment:
This should not be necessary as all columns have the prop attribute. The columns are initiated during `ngOnInit`, I suspect the initialization hasn't worked out correctly than through your recent change.

Reply:
Thanks for reviewing!
The change here is used to skip the ["expand caret column"](https://github.com/ceph/ceph/blob/713ef3c7e762ef9a07171570426c6462d3ecf2d6/src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts#L388-L405), which was introduced in https://github.com/ceph/ceph/pull/32747

The column has no property.  A more obvious code is to simply skip the first column, but I don't like because it will break if we change the location of expand column.
```
expect(_.slice(component.columns, 1).every((column) => Boolean(column.prop))).toBeTruthy()
```